scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/edf/EDFPolicyExtended.java

```diff
@@ -66,10 +63,36 @@
 
     private static final Logger LOGGER = Logger.getLogger(EDFPolicyExtended.class);
 
+    private static final Set<JobId> alreadyNotified = new HashSet<>();
+
+    private synchronized void logIfJobWillMissItsDeadline(List<JobDescriptor> jobs) {
+        Date now = new Date();
+        jobs.stream()
+            .map(jobDescriptor -> ((JobDescriptorImpl) jobDescriptor).getInternal())
+            .filter(job -> job.getJobDeadline().isPresent())
+            .filter(job -> job.getJobInfo().getStartTime() >= 0)
+            .filter(job -> {
+                Date deadline = getEffectiveDeadline(job);
+                Date expeFinish = getEffectiveExpectedExecutionTime(job, now);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "expeFinish" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)